p2p/p2p.go

```diff
@@ -66,12 +66,46 @@ func New(addr, version, peers, privKeyStr string, feederNode bool, bc *blockchai
 		return nil, err
 	}
 
+	var publicMultiAddr multiaddr.Multiaddr = nil
```

Comment:
you don't need to assign nil explicitly because `multiaddr.Multiaddr` is interface and zero value is nil

Reply:
oh, so just become something like this?

```suggestion
	var publicMultiAddr multiaddr.Multiaddr
```

Reply:
yep, exactly